src/_data/glossary.yml

```diff
@@ -1,3 +1,17 @@
+- term: "Assist"
+  short_description: |-
+    An automated, local code edit targeted at common improvements and conventions
+    based on the structure of the code.
+  related_links:
+    - text: "Providing Quick Assists"
```

Comment:
```suggestion
    - text: "Providing quick assists"
```

Reply:
I wouldn't include any of the references to `analysis_server` docs. I think these glossary entries should be aimed at users of the tools rather than contributors to the tools. Knowing how these features are implemented doesn't help users.

Reply:
Dropping these makes sense to me!